_posts/2018-05-02-cockpit-167.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Cockpit 167
+author: pitti
+date: 2018-05-02 14:00
+tags: cockpit linux
+slug: cockpit-167
+category: release
+summary: Cockpit with new Firewall UI
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 167.
+
+### Networking: Add Firewall Configuration
+
+The Networking page now shows the status of
+[firewalld](https://www.firewalld.org/) and has a new "Allowed Services"
+details page where individual services (ports) can be configured.
```

Comment:
This sentence is OK, but is a bit of a run-on. It would be more optimized by splitting it into two sentences.

Perhaps something like:

> The Networking page now displays firewall status using firewalld. Additionally, services and ports can be configured using "Allowed Services" detail pages.

Reply:
Thanks for your suggestions! Fixed.